server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/resources/cucumber/DownloadEndpoint.feature

```diff
@@ -86,7 +86,7 @@ Feature: Download endpoint
     Then the user should receive a not found response
 
   Scenario: An authenticated user should have access to a shared attachment
-    Given "usera@domain.tld" shares its mailbox "INBOX" with "userb@domain.tld"
+    Given "usera@domain.tld" shares (his|her) mailbox "INBOX" with "userb@domain.tld" with "lr" rights
```

Comment:
come on, it doesn't make sense : is usera a boy or a girl ? please choose !

Reply:
Ok, I did not get this one.